features/functions.yml

```diff
@@ -0,0 +1,45 @@
+name: Functions
+description: "Functions are series of statements that can be called and executed and return a value. The `function` keyword (as in `function () { }`) and arrow (`=>`) expression create functions. The JavaScript functions protocol includes default and rest parameters and binding to `this`."
```

Comment:
I'm fine with keeping the additional stuff (default, rest, binding), though I don't think we've been consistent about naming everything that a feature contains all the time. In fact, naming everything will likely, in some cases, go against our 300 max char rule and our intent to speak in developer-friendly terms.

I would suggest removing "and executed" and simply have "that can be called and return a value". Running a function and executing a function mean the same thing to me. And I much prefer the word "run" in this case.

Reply:
```suggestion
description: "Functions are series of statements that can be called and return a value. The `function` keyword (as in `function () { }`) and arrow (`=>`) expression create functions. The JavaScript functions protocol includes default and rest parameters and binding to `this`."
```